.github/workflows/release.yml

```diff
@@ -79,30 +81,10 @@ jobs:
         run: poetry build
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
-      - name: Release Changelog Builder
-        uses: mikepenz/release-changelog-builder-action@v4 # https://github.com/marketplace/actions/release-changelog-builder
-        id: gen_note
-        with:
-          configuration: ".changelog.json"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Create Release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: softprops/action-gh-release@v0.1.15 # https://github.com/marketplace/actions/gh-release
```

Comment:
**Suggestion:** Consider using a more recent version of the 'softprops/action-gh-release' action for better features and bug fixes.
```suggestion
        uses: softprops/action-gh-release@v1.1.5
```